src/HW4/StudentTest.java

```diff
@@ -0,0 +1,40 @@
+package HW4;
+
+public class StudentTest {
+    public static void main(String[] args) {
+
+        Student student1 = new Student();
```

Comment:
It's better to use constructor with parameters Student(String lastName, double mathMark, double economicMark,
double foreignLangMark)